service/templates/distros.html

```diff
@@ -365,14 +377,14 @@ <h2 style="display:inline-block; padding-right:15px">Configure
               </div>
               <div ng-show="activeDistro.provider.startsWith('ec2-spot')">
                 <label class="distro-label">Bid Price:</label>
-                <input ng-readonly="readOnly" ng-required="activeDistro.provider.startsWith('ec2-spot')" name="bidPrice"
+                <input ng-readonly="readOnly" name="bidPrice"
```

Comment:
Ah removing the the one on like 364 and 387 was (because it's redundant given the whole block has that validation) but I didn't realize this one was specific for spot.